admin/services/check-form.service.js

```diff
@@ -225,6 +225,92 @@ const checkFormService = {
       return line.trim()
     }).filter(Boolean)
     return result.length === config.LINES_PER_CHECK_FORM
+  },
+
+  /**
+   * Get unassigned forms for selected check window.
+   * @param checkWindowAssignedForms
+   * @returns {Promise<*>}
+   */
+  getUnassignedFormsForCheckWindow: async (checkWindowAssignedForms) => {
+    let checkFormData
+    let checkFormList = []
+
+    if (!checkWindowAssignedForms) {
+      return false
+    }
+
+    checkFormData = await checkFormDataService.fetchSortedActiveForms({}, 'name', 'asc')
+
+    if (checkWindowAssignedForms && checkFormData) {
+      checkFormData.map((form) => {
+        if (checkWindowAssignedForms.filter(item => item === form._id).length < 1) {
+          checkFormList.push({
+            '_id': form._id,
+            'name': form.name
+          })
+        }
+      })
+    }
+
+    return checkFormList
+  },
+
+  /**
+   * Get assigned forms for selected check window.
+   * @param checkWindowAssignedForms
+   * @returns {Promise<*>}
+   */
+  getAssignedFormsForCheckWindow: async (checkWindowAssignedForms) => {
+    let checkFormData
+    let checkFormList = []
+
+    if (!checkWindowAssignedForms) {
+      return false
```

Comment:
Again here, would `[]` be better for the return?

Reply:
Same as above, views needs editing too.